src/core/MOM.F90

```diff
@@ -2143,6 +2161,13 @@ subroutine initialize_MOM(Time, Time_init, param_file, dirs, CS, restart_CSp, &
 
   call callTree_waypoint("MOM parameters read (initialize_MOM)")
 
+  call get_param(param_file, "MOM", "HOMOGENIZE_FORCINGS", CS%homogenize_forcings, &
+                 "If True, homogenize the forces and fluxes.", default=.false.)
+  call get_param(param_file, "MOM", "UPDATE_USTAR",CS%update_ustar, &
```

Comment:
Because UPDATE_USTAR is only relevant if HOMOGENIZE_FORCINGS is true, the get_param call for UPDATE_USTAR should include the argument `do_not_log=.not.CS%homogenize_forcings`.

Reply:
Thanks @Hallberg-NOAA, I have made the change as you suggest.